configuration/conf/config.properties

```diff
@@ -137,11 +137,17 @@
 # 传感器名称前缀
 # SENSOR_NAME_PREFIX=s_
 
-# 标签名的前缀
-# TAG_NAME_PREFIX=tag_
+# 标签名数量
+# TAG_NUMBER=0
 
-# 定义设备标签值，所有设备相同，使用逗号分开，目前支持iotdb-0.12, iotdb-0.13, influxdb-2.x, timescaledb, tdengine
-# DEVICE_TAGS=
+# 标签名前缀
+# TAG_KEY_PREFIX=tag_
+
+# 标签值前缀
+# TAG_VALUE_PREFIX=value_
+
+# 每个标签值的取值种数，使用逗号分开，设置个数应与标签名数量相等。例如TAG_NUMBER=2，TAG_VALUE_CARDINALITY=50,10，说明第一个标签名可能有50种值，第二个标签名可能有10种值
```

Comment:
Can you further describe what the complete path might look like in this scenario? Would the path be root.test.device.value_10.value_5.sensor?